components/bl-rating-reviews/src/views.js

```diff
@@ -0,0 +1,161 @@
+import { IconRow } from './subcomponents/icon-row';
+import { ProgressBar } from './subcomponents/progress-bar';
+import { calculatePercent } from './helpers';
+import { icons } from './icons';
+
+const { EmptyStar, FilledStar, EmptyHeart, FilledHeart } = icons;
+
+const OneColorReview = (
+  { width, average, maxValue, color, iconSize, totalReviews, reviewData, localizations, eventHandlers }
+) => {
+  return (
+    <div className="oneColorReview" style={{ 'width': width, 'color': color }}>
+      <div className="top-container">
+        <div className="average">
+          <span className="average-value"> {average} {localizations.averageText} {maxValue} </span>
+        </div>
+        <div className="total-reviews">
+          <IconRow
+            Empty={ EmptyStar }
+            Filled={ FilledStar }
```

Comment:
Why we pass `EmptyStar `and `FilledStar `components as a property and don't import it in icon-row.js file ?